tests/components/lupusec/test_config_flow.py

```diff
@@ -0,0 +1,94 @@
+""""Unit tests for the Lupusec config flow."""
+
+import pytest
+from voluptuous.error import MultipleInvalid
+
+from homeassistant import config_entries
+from homeassistant.components.lupusec.config_flow import (
+    is_valid_host,
+    validate_configuration,
+)
+from homeassistant.components.lupusec.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import RESULT_TYPE_CREATE_ENTRY, RESULT_TYPE_FORM
+
+
+async def test_form_empty_input(hass: HomeAssistant) -> None:
+    """Test handling empty user input."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["errors"] == {}
+
+    try:
+        await hass.config_entries.flow.async_configure(result["flow_id"], {})
+        pytest.fail("Expected error not raised")
+    except MultipleInvalid as e:
+        # Check if the error contains the expected missing key
+        assert any(
+            "required key not provided @ data['username']" in str(err)
+            for err in e.errors
+        )
```

Comment:
I think we already have enough tests in core that cover this, so you can rely on the fact that the data schema is followed

Reply:
Removed.